docs/content-services/interfaces/content-metadata-custom-panel.interface.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: Content Metadata Custom Panel interface
+Added: v6.4.0
+Status: Active
+Last reviewed: 2023-10-06
+---
+
+# [Content Metadata Custom Panel interface](../../../lib/content-services/src/lib/content-metadata//interfaces/content-metadata-custom-panel.interface.ts "Defined in content-metadata-custom-panel.interface.ts")
```

Comment:
this path is probably incorrect as redirection does not work

Reply:
Can you check again please? I double checked and it works for me

Reply:
@MichalKinas it still does not work for me but I know why, there are double // in the path before interfaces word: 
<img width="512" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/fe13d8ba-f21c-4af9-86cf-c14cf3447487">

If you make it to single then it is proper path.

Reply:
Right, fixed